tasks/install.yml

```diff
@@ -28,13 +28,13 @@
     state: present
   when:
     - openvpn_use_ldap
-    - ansible_distribution == "CentOS" and ansible_distribution_major_version != "8"
+#     - ansible_distribution == "CentOS" and ansible_distribution_major_version != "8"
```

Comment:
I think this was accidentally commented out. Probably meant to change it to the distribution type?